docs/dashboards/frequently-asked-questions.md

```diff
@@ -0,0 +1,120 @@
+[Here is the demo](https://jsfiddle.net/gh/get/library/pure/highcharts/highcharts/samples/dashboard/demos/dashboard-minimal).
+
+* * *
+
+How to connect component to a cell?
+----------------------------------
+Each cell has a field called id. The same id must be passed in the component config to the `cell` field. Example configuration of component and cell:
+
+```js
+    gui: {
+        enabled: true,
+        layouts: [{
+            id: 'layout-1',
+            rows: [{
+                cells: [{
+                    id: 'dashboard-col-0'
+                }]
+            }]
+        }]
+    },
+    components: [
+        {
+            cell: 'dashboard-col-0',
+            type: 'Highcharts',
+            chartOptions: {
+                chart: {
+                    type: 'pie'
+                },
+                series: [{data: [1,2,3]}]
+            },
+        }]
+```
+
+[Here is the demo](https://jsfiddle.net/gh/get/library/pure/highcharts/highcharts/samples/dashboard/demos/component-highcharts).
+
+* * *
+
+How to synchronize the components?
+-----------------------------
+To synchronize components you have to specify which event you want to synchronize between each component, as well as they have to use the same store.
+
+Example of synchronized components
+
+```js
+components: [{
+    store,
+    sync: {
+        visibility: true,
+        tooltip: true,
+        selection: true
+    },
+    cell: 'dashboard-col-0',
+    isResizable: true,
+    type: 'Highcharts',
+    tableAxisMap: {
+        Food: 'x',
+        'Vitamin A': 'value'
+    },
+    chartOptions: {
+        chart: {
+            type: 'pie'
+        }
+    },
+}, {
+    cell: 'dashboard-col-1',
+    store,
+    sync: {
+        visibility: true,
+        tooltip: true,
+        selection: true
+    },
+    type: 'Highcharts',
+    tableAxisMap: {
+        Food: 'x',
+        'Vitamin A': 'y'
+    },
+    chartOptions: {
+        xAxis: {
+            type: 'category'
+        },
+        chart: {
+            animation: false,
+            type: 'column'
+        }
+    }
+}]
+```
+
+
+
+You can check how this synchronization works in our [minimal dashboards demo](https://jsfiddle.net/gh/get/library/pure/highcharts/highcharts/samples/dashboard/demos/dashboard-minimal).
+
+* * *
+
+What browsers are supported?
+---------------
+The layout of our library is based on flexbox, so in general all browsers, which support flexbox should also work fine with Highcharts Dashboards.
+In particular, those are Chrome, Edge, Firefox, and Safari.
+
+* * *
+
+What are the synchronization events available in Highcharts Dashboards?
+-----------------------------------------------------------------------
+You can check how this synchronization works in our [minimal dashboards demo](https://jsfiddle.net/gh/get/library/pure/highcharts/highcharts/samples/dashboard/demos/dashboard-minimal).
+
+The events, that can be synchronized between components are:
```

Comment:
I would also add that you only can use that sync between chart-chart because DataGrid has only a `tooltip` implemented.